sys/net/crosslayer/ng_pktbuf_static/ng_pktbuf_static.c

```diff
@@ -197,7 +197,7 @@ void ng_pktbuf_hold(ng_pktsnip_t *pkt, unsigned int num)
 void ng_pktbuf_release(ng_pktsnip_t *pkt)
 {
     mutex_lock(&_mutex);
-    while (pkt) {
+    while (_pktbuf_contains(pkt)) {
```

Comment:
```diff
     while (pkt) {
+        ng_pktsnip_t *tmp;
+        assert(_pktbuf_contains(pkt));
+        tmp = pkt->next;
-        ng_pktsnip_t *tmp = pkt->next;
```

And please leave a comment if you make a change, otherwise I won't notice ;-)